src/xrGame/reward_event_handler.h

```diff
@@ -11,7 +11,7 @@ class reward_event_handler
 {
 public:
     reward_event_handler(game_state_accumulator* pstate) : m_player_state_accum(pstate) {}
-    ~reward_event_handler(){};
+    virtual ~reward_event_handler(){};
```

Comment:
Let's let the compiler generate everything else.
```suggestion
    virtual ~reward_event_handler() = default;
```

Reply:
Ok, I just tried to limit the changes to the required minimum.